### PR TITLE
Brainvision units

### PR DESCRIPTION
**Pull request by jaeilepp**
Take over from https://github.com/mne-tools/mne-python/pull/3153.
Closes https://github.com/mne-tools/mne-python/issues/3126
I added ``misc='auto'`` option to brainvision reader. LMKWDYT

### CODE REVIEW

File: mne/io/brainvision/tests/test_brainvision.py [hunk 1]
```diff
@@ -80,6 +82,8 @@ def test_brainvision_data():
             assert_equal(ch['kind'], FIFF.FIFFV_EOG_CH)
         elif ch['ch_name'] == 'STI 014':
             assert_equal(ch['kind'], FIFF.FIFFV_STIM_CH)
+        elif ch['ch_name'] in ('CP6', 'HL', 'HR', 'Vb', 'ReRef'):
+            assert_equal(ch['kind'], FIFF.FIFFV_MISC_CH)
```
Comment (Eric89GXL) [inline mne/io/brainvision/tests/test_brainvision.py]: Can you check that `ch['unit']` is correct, too?

**Comment by Eric89GXL** [comment]
Otherwise LGTM

**Comment by jaeilepp** [comment]
The misc channels were actually set to volts before. I changed it so that they're set to none or celcius.

File: mne/io/brainvision/brainvision.py [hunk 2]
```diff
@@ -434,7 +448,10 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
         elif ch_name in misc or idx in misc or idx - nchan in misc:
             kind = FIFF.FIFFV_MISC_CH
             coil_type = FIFF.FIFFV_COIL_NONE
-            unit = FIFF.FIFF_UNIT_V
+            if ch_name in misc_chs.keys():
```
Comment (agramfort) [inline mne/io/brainvision/brainvision.py]: ch_name in misc_chs

is enough

**Comment by agramfort** [comment]
besides LGTM

would I get an error if a channel has a weird unit and I did not set misc to "auto"?

**Comment by jaeilepp** [comment]
> would I get an error if a channel has a weird unit and I did not set misc to "auto"?

No. But maybe we should set it to 'auto' by default wdyt?

**Comment by agramfort** [comment]
Yes set it to auto by default 






On Sun, May 29, 2016 at 11:05 PM -0700, "jaeilepp" <notifications@github.com> wrote:













would I get an error if a channel has a weird unit and I did not set misc to "auto"?




No. But maybe we should set it to 'auto' by default wdyt?

File: mne/io/brainvision/brainvision.py [hunk 3]
```diff
@@ -38,10 +38,11 @@ class RawBrainVision(_BaseRaw):
         Names of channels or list of indices that should be designated
         EOG channels. Values should correspond to the vhdr file.
         Default is ``('HEOGL', 'HEOGR', 'VEOGb')``.
-    misc : list or tuple
+misc : list or tuple of str | 'auto'
```
Comment (agramfort) [inline mne/io/brainvision/brainvision.py]: bad indent

**Comment by agramfort** [comment]
also don't forget to update what's new

then +1 for merge

File: doc/whats_new.rst [hunk 4]
```diff
@@ -1569,3 +1571,5 @@ of commits):
 .. _Natalie Klein: http://www.stat.cmu.edu/people/students/neklein
 
 .. _Jon Houck: http://www.unm.edu/~jhouck/
+
+.. _Pablo-Arias: https://github.com/Pablo-Arias
```
Comment (jaeilepp) [inline doc/whats_new.rst]: @Pablo-Arias I added your github account here. Feel free to update it to something else if you feel like it.

File: mne/io/brainvision/brainvision.py [hunk 5]
```diff
@@ -70,7 +71,7 @@ class RawBrainVision(_BaseRaw):
     """
     @verbose
     def __init__(self, vhdr_fname, montage=None,
-                 eog=('HEOGL', 'HEOGR', 'VEOGb'), misc=(),
+                 eog=('HEOGL', 'HEOGR', 'VEOGb'), misc='auto',
```
Comment (Eric89GXL) [inline mne/io/brainvision/brainvision.py]: this is an API change so it needs to be noted in `whats_new` that now channels with non-`uV` units will be turned to `misc` (if that is indeed what happens)

**Comment by jaeilepp** [comment]
> this is an API change so it needs to be noted in whats_new that now channels with non-uV units will be turned to misc (if that is indeed what happens)

Done.

**Comment by agramfort** [comment]
thanks @jaeilepp !

**Pull request merged**